### PR TITLE
Added Uno to Swags

### PR DESCRIPTION
**Pull request by audrius59**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added Uno swag

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -314,6 +314,15 @@
...
     "tags": ["hacktoberfest", "expired"]
   },
+  {
+    "name": "Uno",
+    "difficulty": "easy",
+    "description": "Register for the Hacktoberfest® event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
+    "reference": "https://platform.uno/uno-is-joining-hacktoberfest-2019/",
+    "image": "https://s3.amazonaws.com/uno-website-assets/wp-content/uploads/2019/08/21141002/diagram-991x1024.png",
+    "dateAdded": "2019-09-25T13:05:05.000Z",
+    "tags": ["clothing"]
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "tags": ["clothing", "hacktoberfest"]
```

File: data.json [hunk 2]
```diff
@@ -314,6 +314,15 @@
     "dateAdded": "2018-10-20T12:48:46.000Z",
     "tags": ["hacktoberfest", "expired"]
   },
+  {
+    "name": "Uno",
+    "difficulty": "easy",
+    "description": "Register for the Hacktoberfest® event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "Register for the Hacktoberfest event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
```
Not sure about the registered trademark symbol.

File: data.json [hunk 3]
```diff
@@ -333,6 +333,14 @@
...
   },
   {
+    "name": "Uno",
+    "difficulty": "easy",
+    "description": "Register for the Hacktoberfest event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
+    "reference": "https://platform.uno/uno-is-joining-hacktoberfest-2019/",
+    "image": "https://s3.amazonaws.com/uno-website-assets/wp-content/uploads/2019/08/21141002/diagram-991x1024.png",
+    "dateAdded": "2019-09-25T13:05:05.000Z",
+    "tags": ["clothing", "hacktoberfest"]
+  },
```
Comment (aslafy-z) [inline data.json]:
```suggestion
  },
  {
```

**Comment by audrius59** [comment]
Sure, as @aslafy-z suggest anyone's welcome 🙂

**Pull request merged**